src/components/BorrowerProfile/LendCta.vue

```diff
@@ -438,6 +442,12 @@ export default {
 				loanId: this.loanId,
 			}).then(() => {
 				this.isAdding = false;
+				if (this.isCompleteLoanActive) {
+					// eslint-disable-next-line max-len
+					this.$kvTrackEvent('Borrower profile', 'Complete loan', 'click-amount-left-cta', this.loanId, this.selectedOption);
+				} else {
+					this.$kvTrackEvent('Lending', 'Add to basket', this.ctaButtonText);
```

Comment:
This event should fire for ALL add-to-basket actions regardless of loan share size.

Reply:
Nice, fixed!